spec/requests/posts_request_spec.rb

```diff
@@ -10,8 +10,8 @@
 
   describe 'GET #index' do
     it 'displays all posts' do
-      get "/users/#{user.id}/posts"
-
+      # get "/users/#{user.id}/posts"
+      visit(user_posts_path(user))
       expect(response).to have_http_status(200)
       expect(response.body).to include(user.name)
       expect(response.body).to include(subject.title)
```

Comment:
- [ ] Please make sure you have all your test passing :pray: Some of your test written already are falling since you modified them to get the views test working. However, you should have all your test passing to make sure the application is working completely. :+1: 

  ![image](https://user-images.githubusercontent.com/92303891/226867381-a37b7fa7-0bb1-4fdd-8039-63813de41278.png)

Reply:
Hey  @Owusu-Desmond , thank you for your thorough feedback. I have looked into the test examples there is no issue with it but it is related to the privilege. Can you please help me? Its blocking me from progressing. ☺️